src/views/common.tsx

```diff
@@ -55,9 +57,15 @@ export const FormButton = (props: FormButtonProps): JSX.Element => (
  * @returns JSX - Sidarbar
  */
 const SideBar = ({ activePage }: { activePage: PageProps['activePage'] }): JSX.Element => {
+  const title = new Env().get('API_SWAGGER_TITLE').slice(0, 1)
```

Comment:
don't pull in `env` like this. Use `container.resolve`

Reply:
thanks for a flag up, updated to use `const env = container.resolve(Env)`